gems/pending/util/duplicate_blocker/dedup_handler.rb

```diff
@@ -63,8 +68,8 @@ def handle(method, *args)
       key = key_generator.call(method, *args)
       desc = descriptor.call(method, *args)
 
-      entry = update_history(key, desc)
-      tripped?(entry) ? on_blocking_call(desc) : method[*args]
+      entry = update_history(key.hash, desc)
+      tripped?(entry, desc) ? on_blocking_call(desc) : method[*args]
```

Comment:
I did some test comparing the memory usage with or without saving description. Assume the a provider generates an event every half second, and we store all events in the past 5 minutes. VmPoweredOffEvent is used as an example. The description string converting from the event is about 580 characters. It saves 640KB memory if we do not save the descriptions in the hash.